client/js/analysis.js

```diff
@@ -0,0 +1,27 @@
+
+
+window.onload = function() {
+    // Listen for on submit click
+    let formSubmit = document.getElementById("formSubmit");
+    formSubmit.addEventListener("click", fetchAndShowResponse, false);
+}
+
+/**
+ * Grabs response from server
+ */ 
+async function fetchAndShowResponse() {
+    const response = await fetch('/api/musix');
+    const value = await response.json();
+    changeInnerText("response", value);
+    return value;
+}
+
+/**
+ * Change the contents of div
+ * @param elId the id of the element 
+ * @param value the value to insert as innerText
+ */
+function changeInnerText(elId, value) {
```

Comment:
Consider renaming this "setInnerText", and updating the function comment to be more generic since the element isn't necessarily a div (this helper could later be used for something other than the "response" div).

Reply:
+1. I'd also consider getting rid of this helper entirely since it makes the parent function less easy to read. In general, you probs don't need a helper function if it's under 3 lines of code (unless they're complicated lines of code).